Source/OxyPlot/PlotModel/PlotModel.cs

```diff
@@ -674,7 +637,7 @@ public OxyColor GetDefaultColor()
         /// <returns>The next default line style.</returns>
         public LineStyle GetDefaultLineStyle()
         {
-            return (LineStyle)((this.currentColorIndex / this.DefaultColors.Count) % (int)LineStyle.None);
+            return (LineStyle)(this.currentColorIndex / this.DefaultColors.Count % (int)LineStyle.None);
```

Comment:
I don't like this at all: it reads badly and almost no-one knows the precedence rules for `%`.

Reply:
Again, it's something where I'd prefer to leave it to the programmer to decide. I don't mind removing parens for `x * y + z` etc. because it's well known, but for less well known operators I'd prefer it to be discretionary.

Reply:
We actually have the rule set up so it's up to the programmer (we have it on `silent`), so it will not give a suggestion when 'unnecessary' braces are added. This was again a case of the VS refactoring being somewhat eager